wifiphisher/common/phishingpage.py

```diff
@@ -366,6 +366,9 @@ def find_user_templates(self):
                 if self.search_template(name):
                     # add it to the list
                     local_templates.append(name)
+                    print "[" + constants.G + "!" + constants.W + "] Found template in: " + name
+                else:
+                    print "[" + constants.R + "!" + constants.W + "] No template found in: " + name
```

Comment:
Maybe we can print more specifically with message: `.config or .html are not found in `

Reply:
Sure, but should we return a string like "config file not found"?

Reply:
how about `configuration files are not found in `

Reply:
Uhm, alright